core/src/main/resources/reference.conf

```diff
@@ -1578,6 +1578,13 @@ datastax-java-driver {
         # Valid for: Micrometer.
         // slo = [ 100 milliseconds, 500 milliseconds, 1 second ]
         
+        # An optional list of percentiles to be published by Micrometer.
+        #
+        # If defined, the histogram is guaranteed to contain these percentiles alongside other
+        # buckets used to generate aggregable percentile approximations.
```

Comment:
We should probably be very explicit here in saying that these percentiles can **_NOT_** be aggregated across multiple nodes in the way that percentile histograms can.

Reply:
thanks. added comment